connectors/src/connectors/snowflake/lib/content_nodes.ts

```diff
@@ -0,0 +1,89 @@
+import type { ConnectorPermission, ContentNode } from "@dust-tt/types";
+
+/**
+ * Some databases and schemas are not useful to show in the content tree.
+ * We exclude them here.
+ */
+export const EXCLUDE_DATABASES = ["SNOWFLAKE"];
+export const EXCLUDE_SCHEMAS = ["INFORMATION_SCHEMA"];
+
+/**
+ * 3 types of nodes in a remote database content tree:
+ * - database: internalId is the database name: "database_name"
+ * - schema: internalId is the schema name: "database_name.schema_name"
```

Comment:
IMO having "is the schema name" in the comment makes it a tiny bit more confusing